docs/source/models.rst

```diff
@@ -113,80 +113,28 @@ for saving and loading models to and from this format. The format is self-contai
 that it includes all the information necessary to load and use a model. Dependencies
 are stored either directly with the model or referenced via Conda environment.
 
+Many of MLflow's model persistence modules, such as :mod:`mlflow.sklearn`, :mod:`mlflow.keras`,
+and :mod:`mlflow.pytorch`, produce models with the ``python_function`` (``pyfunc``) flavor. This
+means that they adhere to the ``mlflow.pyfunc`` filesystem format and can be interpreted as
+generic Python classes that implement the specified :ref:`inference API <pyfunc-inference-api>`.
+Therefore, any tool that operates on these ``pyfunc`` classes can operate on any MLflow model
+containing the ``pyfunc`` flavor, regardless of which persistence module or framework was used to
+produce the model. *This interoperability is very powerful because it allows any Python model to be
+productionized in a variety of environments*.
+
 The convention for ``python_function`` models is to have a ``predict`` method or function with the following
 signature:
 
 .. code:: python
 
-    predict(data: pandas.DataFrame) -> [pandas.DataFrame | numpy.array]
+    predict(data: pandas.DataFrame) -> numpy.ndarray | pandas.Series | pandas.DataFrame
```

Comment:
I think you need the box brackets here, to suggest one of.

```
    predict(data: pandas.DataFrame) -> [numpy.ndarray | pandas.Series | pandas.DataFrame]
```

Looks like this on [mlflow.org/.../models.html](https://mlflow.org/docs/latest/models.html) documentation

![image](https://user-images.githubusercontent.com/6900999/51003310-0c6f4680-14eb-11e9-98b9-3762d7b1e956.png)

Reply:
Good catch. I've added brackets to fix the documentation.